shared/fs/index.js

```diff
@@ -19,30 +19,34 @@ type FolderProps = {
   routePath: I.List<string>,
 }
 
+export const wrapRow = (rowContent, key: string) => (
```

Comment:
don't make helper functions that return a component, just make it a component
```
const WrapRow = ({children, id}) => 
```

Reply:
Sure; Does this have something to do with re-rendering?

Reply:
no really its just if its a function that takes in a component and returns a component, then it itself is a component. we don't need 2 forms of this

Reply:
Got it thanks!